docs/tutorials/zoom-activity-dashboard.md

```diff
@@ -0,0 +1,277 @@
+
+Choosing Zoom as **source type** will cause Airbyte to display the configuration parameters needed to set up the Zoom source.
+
+![](../.gitbook/assets/02_setting-zoom-connector-name.png)
+
+The Zoom connector for Airbyte requires you to provide it with a Zoom JWT token. Let’s take a detour and look at how to obtain one from Zoom. 
+
+### Obtaining a Zoom JWT Token
+
+To obtain a Zoom JWT Token, login to your Zoom account and go to the [Zoom Marketplace](https://marketplace.zoom.us/). If this is your first time in the marketplace, you will need to agree to the Zoom’s marketplace terms of use. 
+
+Once you are in, you need to click on the **Develop** dropdown and then click on **Build App.** 
+
+![](../.gitbook/assets/03_click.png)
+
+Clicking on **Build App** for the first time will display a modal for you to accept the Zoom’s API license and terms of use. Do accept if you agree and you will be presented with the below screen.
+
+![](../.gitbook/assets/04_zoom-marketplace-build-screen.png)
+
+Select **JWT** as the app you want to build and click on the **Create** button on the card. You will be presented with a modal to enter the app name; type in `airbyte-zoom`.
+
+![](../.gitbook/assets/05_app-name-modal.png)
+
+Next, click on the **Create** button on the modal.
+
+You will then be taken to the **App Information** page of the app you just created. Fill in the required information \(at the very least\).
+
+![](../.gitbook/assets/06_app-information.png)
+
+After filling in the needed information, click on the **Continue** button. You will be taken to the **App Credentials** page. Here, click on the **View JWT Token** dropdown.
+
+![](../.gitbook/assets/07_view-jwt-token.png)
+
+There you can set the expiration time of the token \(we will leave the default 90 minutes\), and then you click on the **Copy** button of the **JWT Token**.
+
+After copying it, click on the **Continue** button.
+
+![](../.gitbook/assets/08_activate-webhook.png)
+
+You will be taken to a screen to activate **Event Subscriptions**. Just leave it as is, as we won’t be needing Webhooks. Click on **Continue**, and your app should be marked as activated. 
+
+### Connecting Zoom on Airbyte
+
+So let’s go back to the Airbyte web UI and provide it with the JWT token we copied from our Zoom app.
+
+Now click on the **Set up source** button. You will see the below success message when the connection is made successfully. 
+
+![](../.gitbook/assets/09_setup-successful.png)
+
+And you will be taken to the page to add your destination.
+
+### Connecting PostgreSQL on Airbyte
+
+![](../.gitbook/assets/10_destination.png)
+
+For our destination, we will be using a PostgreSQL, since Tableau supports PostgreSQL as a data source. Click on the **add destination** button, and then in the drop down click on **+ add a new destination**.  In the page that presents itself, add the destination name and choose the Postgres destination.
```

Comment:
```suggestion
For our destination, we will be using a PostgreSQL database, since Tableau supports PostgreSQL as a data source. Click on the **add destination** button, and then in the drop down click on **+ add a new destination**.  In the page that presents itself, add the destination name and choose the Postgres destination.
```